tests/unit/test_display.py

```diff
@@ -0,0 +1,60 @@
+import unittest
```

Comment:
Since we don't have a folder for additional integration tests yet, let's put this file directly under `tests` remove the intermediate folder `unit` for now.

Reply:
ok